.github/workflows/windows-compile.yml

```diff
@@ -0,0 +1,29 @@
+# Platform specific Windows test. See verible-ci for main continuous integration.
+name: macos-compile
+
+on:
+  push:
+    branches:
+    - master
+    - main
+  pull_request:
+    branches:
+    - master
+    - main
+
+jobs:
+  WindowsBuild:
+    runs-on: windows-latest
+    steps:
+    - name: Install deps
+      run: |
+        choco install winflexbison
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Build flex/bison files
+      run: |
+        bazel build //common/analysis:command_file_lex
```

Comment:
I'd probably add a comment saying that these are the first targets ported to work on Windows until Verible is fully platform independent.